### PR TITLE
Logger

### PR DESCRIPTION
**Pull request by milansky**
Basic logger that takes a string argument, need to work out on other values and arguments.
This will suffice for now instead of writing cout for debugging.

### CODE REVIEW

File: src/glfw-sandbox/logger.cpp [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+#include <iostream>
+#include <string>
+#include "spdlog/spdlog.h"
+
+// work in progres, adding some basic stuff at the moment
+// https://github.com/gabime/spdlog
+
+
+
+void logger::trace(std::string text) {
```
Comment (petrSchreiber) [inline src/glfw-sandbox/logger.cpp]: In Nico's voice: "Are you sure the method changes internal state of the object?"

Comment (petrSchreiber) [reply src/glfw-sandbox/logger.cpp]: In Nico's voice: "Are you sure the parameter gets changed inside the method?"

Comment (petrSchreiber) [reply src/glfw-sandbox/logger.cpp]: In Nico's voice: "Are you sure it is necessary to perform copy?"

**Review by petrSchreiber — CHANGES_REQUESTED** [review]
Milan, thanks a lot for bringing this important functionality to the engine - I have basically one and only suggestion regarding the wrapper signatures.

**Comment by milansky** [comment]
Function changed, takes constant value and reference applied so its not copied :)

**Review by petrSchreiber — APPROVED** [review]
Thank you very much for taking the suggestions into consideration 👍

**Pull request merged**